<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HomeWindow</class>
 <widget class="QMainWindow" name="HomeWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1143</width>
    <height>739</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Dubai</family>
    <pointsize>24</pointsize>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>25</width>
        <height>250</height>
       </size>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOn</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
      </property>
      <widget class="QWidget" name="menuWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>217</width>
         <height>1200</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>2</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>700</height>
        </size>
       </property>
       <widget class="QWidget" name="verticalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>221</width>
          <height>1015</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>8</number>
         </property>
         <item>
          <widget class="QLabel" name="lblSectionGeneral">
           <property name="text">
            <string>General</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnUniversidad">
           <property name="font">
            <font>
             <family>Segoe UI</family>
             <pointsize>22</pointsize>
             <bold>false</bold>
            </font>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LayoutDirection::LeftToRight</enum>
           </property>
           <property name="text">
            <string>Universidad</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnFacultad">
           <property name="font">
            <font>
             <family>Segoe UI</family>
             <pointsize>22</pointsize>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Facultades</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnEdificio">
           <property name="font">
            <font>
             <family>Segoe UI</family>
             <pointsize>22</pointsize>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Edificios</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnGrado">
           <property name="font">
            <font>
             <family>Segoe UI</family>
             <pointsize>22</pointsize>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Grados</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnDepartamento">
           <property name="font">
            <font>
             <family>Segoe UI</family>
             <pointsize>22</pointsize>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Dptos.</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblSectionAcademico">
           <property name="text">
            <string>Acádemico</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnAlumno">
           <property name="font">
            <font>
             <family>Segoe UI</family>
             <pointsize>22</pointsize>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Alumnos</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnProfesor">
           <property name="font">
            <font>
             <family>Segoe UI</family>
             <pointsize>22</pointsize>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Profesores</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnAsignatura">
           <property name="font">
            <font>
             <family>Segoe UI</family>
             <pointsize>22</pointsize>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Asignaturas</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnClase">
           <property name="font">
            <font>
             <family>Segoe UI</family>
             <pointsize>22</pointsize>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Clases</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnDebate">
           <property name="font">
            <font>
             <family>Segoe UI</family>
             <pointsize>22</pointsize>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Debates</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnPremio">
           <property name="font">
            <font>
             <family>Segoe UI</family>
             <pointsize>22</pointsize>
             <bold>false</bold>
            </font>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LayoutDirection::LeftToRight</enum>
           </property>
           <property name="text">
            <string>Premios</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblSectionInvestigacion">
           <property name="text">
            <string>Investigación</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnInvestigacion">
           <property name="font">
            <font>
             <family>Segoe UI</family>
             <pointsize>22</pointsize>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Gr. Inv.</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnProyecto">
           <property name="font">
            <font>
             <family>Segoe UI</family>
             <pointsize>22</pointsize>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Proyectos</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnSubvencion">
           <property name="font">
            <font>
             <family>Dubai</family>
             <pointsize>22</pointsize>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Subvenciones</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnPublicacion">
           <property name="font">
            <font>
             <family>Segoe UI</family>
             <pointsize>22</pointsize>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Publicaciones</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="page"/>
      <widget class="QWidget" name="page_2"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1143</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
